data.py-dist

```diff
@@ -23,6 +23,9 @@ HOSTS = {
 # Default VM images location
 DEF_VM_URL = 'http://pxe/images/'
 
+# Default PXE server
+DEF_PXE_IP = '10.0.0.2'
+
```

Comment:
Do we want that?
Isn't it better to pass it to the script as an argument?

Reply:
wasn't in the specifications from @stormi 
i think we don't change that variable so often in a network.

Reply:
Given how specific the whole PXE setup is, I don't think we need to have it as a mandatory parameter we'd type each time.